app/lib/job/backend.dart

```diff
@@ -331,6 +332,11 @@ class _Stat {
 
   void add(Job job) {
     _totalCount++;
```

Comment:
Why do we count jobs in the total if they have the wrong runtime version?

Reply:
Sanity check, in case some counter is 0, we can see that it is indeed counted some `total` number of rows, but the filtering didn't match any of it.